src/commands/play.js

```diff
@@ -21,0 +1,20 @@
+const Player = require('../audio/player.js');
+
+module.exports = {
+    name: 'play',
+    run: function(event, args) {
+        const url = args[0];
+
+        if (!event.member.voice.channel) return event.channel.send('❓ 你必須加入一個語音頻道');
+
+        let player;
+        if (!Player.getSendingPlayer(event.guild)) {
+            player = new Player(event, event.guild, event.member.voice.channel);
+            player.init();
+        } else {
+            player = Player.getSendingPlayer(event.guild);

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/gh/blackcatbot/blackcat/pullRequest?prid=8779465)
```suggestion
            player = Player.getSendingPlayer(event.guild);
```